pkg/operator/ceph/cluster/controller.go

```diff
@@ -1173,3 +1184,8 @@ func populateConfigOverrideConfigMap(context *clusterd.Context, namespace string
 
 	return nil
 }
+
+func hasCleanupPolicy(cephCluster *cephv1.CephCluster) bool {
```

Comment:
How about a unit test for this method? It's certainly simple now, just anticipating expanding with other settings in the future where it becomes more useful.